frontend/webpack.parts.js

```diff
@@ -164,6 +164,22 @@ exports.loadImages = ({
   },
 });
 
+exports.loadFiles = ({ include, exclude } = {}) => ({
+  module: {
+    rules: [
+      {
+        test: /\.(xls?m|pdf|PDF|doc?x)$/,
```

Comment:
I imagine since the `PDF` didn't fix the issue we were debugging, we can probably remove it